cmd/process-agent/config_test.go

```diff
@@ -169,9 +169,11 @@ func TestConnectionsCheck(t *testing.T) {
 }
 
 func TestPodCheck(t *testing.T) {
-	cfg := config.Mock(t)
+	config.SetDetectedFeatures(config.FeatureMap{config.Kubernetes: {}})
```

Comment:
Updated this because introducing `defer SetDetectedFeatures(nil)` in `collector_test.go` causes these tests to break.